DigitalLearningSolutions.Data/Models/LearningHubApiClient/ResourceMetadata.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Data.Models.LearningHubApiClient
+{
+    public class ResourceMetadata
+    {
+        public int ResourceId { get; set; }
+
+        public string? Title { get; set; }

```

Comment:
Some of these properties aren't nullable in the Open API response.
Check out the view models in the [Open API repo](https://dev.azure.com/e-LfH/LearningHub/_git/LearningHub.Nhs.OpenApi?path=/LearningHub.Nhs.OpenApi.Models/ViewModels&version=GBCI&_a=contents) (make sure you're on branch `CI` rather than `Main`).

Reply:
Should the view models match the Open API repo view models exactly? The ticket describes these fields as nullable

Reply:
Ah, the ticket might have been made before the final version of the view models.

In terms of nullability, yeah, we want to be as restrictive as possible here - the data we get back will always match the Open API repo view models, so it's unhelpful to tell DLS devs "the Title might be null" when it actually won't.